aws-sqs-source.kamelet.yaml

```
@@ -44,7 +44,7 @@ spec:
         title: AWS Region
         description: The AWS region to connect to (capitalized name)
```

We should also change the description:

```suggestion
        description: The AWS region to connect to
```